MIDAS/src/systems.cpp

```diff
@@ -47,7 +47,8 @@ DECLARE_THREAD(gyroscope, RocketSystems* arg) {
 
 DECLARE_THREAD(high_g, RocketSystems* arg) {
     while (true) {
-        THREAD_SLEEP(16);
+        arg->rocket_state.high_g.update(arg->sensors.high_g.read());
+        THREAD_SLEEP(20);
```

Comment:
match sampling rate as close as possible

Reply:
![image](https://github.com/ISSUIUC/MIDAS-Software/assets/77648585/f1f2f44a-2ddf-4cbc-9778-5a05ac050b24)

The data sheet i found [here](https://cdn.sparkfun.com/assets/6/a/2/c/b/KX134-1211-Specifications-Rev-1.0.pdf), on page 6 says typical sampling rate is 50 Hz. What should I change it to?

Reply:
whoops my bad you are correct :)